metadata.yaml

```diff
@@ -17,7 +22,11 @@ requires:
     interface: kubernetes-cni
     scope: container
 resources:
-  calico:
+  calico-amd64:
```

Comment:
If we do this rename we will lose all past resources. We will not be able to deploy old releases. Lets not do that. Keep the old name.